terraform/variables.tf

```diff
@@ -2,10 +2,6 @@ variable aws_region {
   default = "eu-west-1"
```

Comment:
I think it's best to remove this default. It should be supplied explicitly as part of the environment configuration

Reply:
Removed